spec/plaindate.html

```diff
@@ -946,8 +945,8 @@ <h1>AddISODate ( _year_, _month_, _day_, _years_, _months_, _weeks_, _days_, _ov
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-compareisodate" aoid="CompareISODate">
```

Comment:
I don't think the name of this operation needs to change, it is still comparing an ISO date. We already added ISO to the names of operations such as this in response to reviewer feedback, so we should not change this name back to what it was before that.